src/app/test_executive/epoch_ledger.ml

```diff
@@ -0,0 +1,75 @@
+open Integration_test_lib
+open Mina_base
+
+module Make (Inputs : Intf.Test.Inputs_intf) = struct
+  open Inputs
+  open Engine
+  open Dsl
+
+  open Test_common.Make (Inputs)
+
+  (* TODO: find a way to avoid this type alias (first class module signatures restrictions make this tricky) *)
+  type network = Network.t
+
+  type node = Network.Node.t
+
+  type dsl = Dsl.t
+
+  let fork_config : Runtime_config.Fork_config.t =
+    { previous_state_hash =
+        "3NKSiqFZQmAS12U8qeX4KNo8b4199spwNh7mrSs4Ci1Vacpfix2Q"
+    ; previous_length = 300000
+    ; previous_global_slot = 500000
+    }
+
+
+  let config =
+    let open Test_config in 
+    let staking_accounts : Test_Account.t list =
+      [ { account_name = "node-a-key"; balance = "0"; timing = Untimed }
+      ; { account_name = "node-b-key"; balance = "0"; timing = Untimed }
+      ]
+    in
+    let staking : Test_config.Epoch_data.Data.t =
+      let epoch_seed =
+        Epoch_seed.to_base58_check Snark_params.Tick.Field.(of_int 42)
+      in
+      let epoch_ledger = staking_accounts in
+      { epoch_ledger; epoch_seed }
+    in
+    (* next accounts contains staking accounts, with balances changed, one new account *)
```

Comment:
Accounts look exactly the same as in staking ledger

Reply:
Let me remove this.